crates/e2e/src/subxt_client.rs

```diff
@@ -87,6 +87,7 @@ use subxt::{
     config::{
         DefaultExtrinsicParams,
         ExtrinsicParams,
+        HashFor,
```

Comment:
`Hash` associated type removed from `Config` in https://github.com/paritytech/subxt/commit/21b3f521917871771ba6e1b71b70fb53938e87d5, providing the HashFor type alias to be used instead.